packages/react-components/react-storybook-addon-export-to-sandbox/src/sandbox-scaffold.ts

```diff
@@ -128,16 +128,15 @@ const Vite = {
       scripts: {
         dev: 'vite',
         build: 'tsc && vite build',
-        lint: 'eslint . --ext ts,tsx --report-unused-disable-directives --max-warnings 0',
```

Comment:
why remove this ?

Reply:
We don't install proper deps:

<img width="380" alt="image" src="https://github.com/microsoft/fluentui/assets/14183168/b67a732b-54c7-41f5-a974-6da84ab8ca47">

Reply:
good catch, we can add those then ;)

Reply:
pasting offline convo for better visibility:

there is additional dependency install time concern.

I mean the "install" time wouldn't be increased significantly , but for the "DEMO" purpose I don't think its necessary to have in place ATM. 

On the other hand if someone has a different workflow, like opens demo, starts playing around etc -> if we had griffel rules in place and our fluent v9 pattern rules(none ATM) I think that would be very valuable for consumer to avoid antipatterns etc.

**Conclusion:** lets remove lint command